test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/CharacterTest.scala

```diff
@@ -47,6 +47,25 @@ class CharacterTest {
     assertEquals('9', Character.forDigit(9, 10))
   }
 
+  @Test def charCount(): Unit = {
+    def random(from: Int, to: Int): Int =
+      scala.util.Random.nextInt(to - from + 1) + from
```

Comment:
It is OK to make random tests, but please seed the random generator to get reproducible results.